apps/api/src/services/vote-manager.service.ts

```diff
@@ -177,15 +177,40 @@ export class VoteManager {
       }
 
       // Check if voting is allowed based on competition status
-      const votingAllowed = this.checkCompetitionVotingStatus(competition);
-      if (!votingAllowed) {
+      const votingEnabledStatuses: Array<SelectCompetition["status"]> = [
```

Comment:
since this and the method below both reuse this logic, should we keep the `this.checkCompetitionVotingStatus(...)` method usage? the method returns a boolean, which can be used in the subsequent `if` condition.

(optional: maybe we rename `checkCompetitionVotingStatus` -> `isVotingAllowedByStatus`)